.github/workflows/uploadBinaries.yml

```diff
@@ -99,23 +99,42 @@ jobs:
             -DLLVM_FORCE_ENABLE_STATS=${{ matrix.build_config.stats }} \
             -DCIRCT_RELEASE_TAG_ENABLED=ON \
             -DCIRCT_RELEASE_TAG=${{ github.ref_name }} \
-            -DCMAKE_EXPORT_COMPILE_COMMANDS=OFF
+            -DCMAKE_EXPORT_COMPILE_COMMANDS=OFF \
+            -DCMAKE_INSTALL_PREFIX=`pwd`/../install
           ninja
           ninja check-circt check-circt-unit
+          ninja install
+          cd ..
+
       - name: Display Files
-        run: file build/bin/*
+        run: |
+          file install/*
+          file install/bin/*
+
+      - name: Name Install Directory
+        id: name_dir
+        run: |
+          SANITIZED=$(echo ${{ github.ref_name }} | sed 's/\//-/g')
+          echo "value=$SANITIZED" >> "$GITHUB_OUTPUT"
```

Comment:
The directory used for the packaged up stuff is named after the tag or branch, so if you do a workflow dispatch on branch main, it'll be called main, if you do a tag, it'll be the name of the tag (eg. firtool-1.31.0).

The sanitation was added in case there is a `/` in the name of the branch or tag.

Reply:
Would it be better to use `git describe --tag` here instead of the branch name? (Is the name of the branch ever used?)

Reply:
Yeah I agree that is better, I'll change it.